pss/pss.go

```diff
@@ -52,11 +59,41 @@ func (p *Pss) Send(ctx context.Context, targets [][]byte, topic trojan.Topic, pa
 		return nil, err
 	}
 
+	tag := chunk.NewTag(1, "pss-chunks-tag", 0, false)
```

Comment:
curious: does it matter what `uid` we set for the `tag`?

Reply:
Since there is only one tag per chunk, the number is always going to be the same.
So it does not matter in this case.

Reply:
ok again i think i made  assumptions. So the id does not matter but what does matter is the ability for localstore to persist it and then push sync to pick it up. for this the tag needs to be stored. This is what the Tags object does, so you need to call `tags.Create` which is here https://github.com/ethersphere/swarm/blob/master/chunk/tags.go#L52
instead of NewTag which i dont even get why is exported.

so to  sum up.

send 
- wraps message, 
- create tag
- set tag on chunk
- stores chunk

now there  is 2  options. 
1) if we care about only synced state, then make the call synchronous and call before return

```
tag.Inc(chunk.StateSplit)
tag.DoneSplit(chunk.Address{})
tag.WaitTillDone(chunk.StateSynced)
```

or 
2) we care about the other states too, then  return a monitor channel (capacity 3) and start an async routine, somthing like

```golang
for _, state := range([]chunk.State{chunk.StateStored,chunk.StateSent,chunk.StateSynced}) {
   for {
       n, total, err := tag.State(state)
       if err ==  nil && n == total {
          monitor <- s
          break
       }
   }
}
```